kd/deployment.yaml

```diff
@@ -109,6 +109,8 @@ spec:
               value: /certs/tls.pem
             - name: SERVER_KEY
               value: /certs/tls-key.pem
+            - name: ADD_NGINX_LOCATION_CFG
+              value: 'proxy_read_timeout 99999s; proxy_connect_timeout 60s;'
```

Comment:
Any reason why this is on one line and the others have been split? If it works on two lines I'd prefer that for readibility.

Reply:
Resolved.